Binance.Net/Objects/Brokerage/SubAccountData/BinanceBrokerageAccountInfo.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Newtonsoft.Json;
+
+namespace Binance.Net.Objects.Brokerage.SubAccountData
+{
+    /// <summary>
+    /// Account Info
+    /// </summary>
+    public class BinanceBrokerageAccountInfo
+    {
+        /// <summary>
+        /// Max Maker Commission
+        /// </summary>
+        [JsonProperty("maxMakerCommission")]
```

Comment:
Just a heads up, if the property is called the same and only differs in capitalization it will be parsed fine without a JsonProperty attribute. 

No need to change this now though